lib/api/pulsar/pulsar_namespace_api.dart

```diff
@@ -52,6 +52,138 @@ class PulsarNamespaceApi {
     return BacklogQuotaResp.fromJson(destinationStorageResp);
   }
 
+  static Future<PolicyResp> getPolicy(String host, int port, String tenant, String namespace) async {
```

Comment:
better to put behind `updateBacklogQuota`, make `getBacklogQuota` close to `updateBacklogQuota`